Source/WebCore/workers/service/WorkerSWClientConnection.cpp

```diff
@@ -381,6 +410,148 @@ void WorkerSWClientConnection::getNavigationPreloadState(ServiceWorkerRegistrati
     });
 }
 
+void WorkerSWClientConnection::startBackgroundFetch(ServiceWorkerRegistrationIdentifier registrationIdentifier, const String& backgroundFetchIdentifier, Vector<BackgroundFetchRequest>&& requests, BackgroundFetchOptions&& options, ExceptionOrBackgroundFetchInformationCallback&& callback)
+{
+    uint64_t requestIdentifier = ++m_lastRequestIdentifier;
```

Comment:
Can these be strongly typed?

Reply:
We could but this is not strictly needed since this remains internal to the WorkerSWClientConnection method.